viewsets/static/scripts/add_more_inlines.js

```diff
@@ -1,13 +1,13 @@
 $(function(){
-    $('.form-inline > .add-more').click(function(event){
```

Comment:
.form-inline is from bootstrap, and applies styling that you may not want added.